symbolic-demangle/src/lib.rs

```diff
@@ -113,16 +141,16 @@ fn is_maybe_swift(_ident: &str) -> bool {
 #[cfg(feature = "msvc")]
 fn try_demangle_msvc(ident: &str, opts: DemangleOptions) -> Option<String> {
     use msvc_demangler::DemangleFlags as MsvcFlags;
-    let flags = match opts.format {
-        DemangleFormat::Full => MsvcFlags::COMPLETE,
-        DemangleFormat::Short => {
-            if opts.with_arguments {
-                MsvcFlags::NO_FUNCTION_RETURNS
-            } else {
-                MsvcFlags::NAME_ONLY
-            }
-        }
-    };
+
+    // the flags are bitflags
+    let mut flags = MsvcFlags::COMPLETE;
+    if !opts.return_type {
+        flags |= MsvcFlags::NO_FUNCTION_RETURNS;
+    }
+    if !opts.arguments {
+        // a `NO_ARGUMENTS` flag is there in the code, but commented out
+        flags |= MsvcFlags::NAME_ONLY;
```

Comment:
Will this mean that `arguments(false)` also disables the return type for MSVC?

Reply:
I think so, yes.